src/wallet/rpcwallet.cpp

```diff
@@ -3605,7 +3604,7 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
             "  \"sigsrequired\" : xxxxx        (numeric, optional) Number of signatures required to spend multisig output (only if \"script\" is \"multisig\")\n"
             "  \"pubkey\" : \"publickeyhex\",    (string, optional) The hex value of the raw public key, for single-key addresses (possibly embedded in P2SH or P2WSH)\n"
             "  \"embedded\" : {...},           (object, optional) Information about the address embedded in P2SH or P2WSH, if relevant and known. It includes all getaddressinfo output fields for the embedded address, excluding metadata (\"timestamp\", \"hdkeypath\", \"hdseedid\") and relation to the wallet (\"ismine\", \"iswatchonly\").\n"
-            "  \"iscompressed\" : true|false,  (boolean) If the address is compressed\n"
+            "  \"iscompressed\" : true|false,  (boolean, optional) If the address is compressed\n"
```

Comment:
Shouldn't this say pubkey?

```suggestion
            "  \"iscompressed\" : true|false,  (boolean, optional) If the pubkey is compressed\n"
```